BrainPortal/app/models/remote_command.rb

```diff
@@ -100,7 +100,11 @@ class RemoteCommand < RestrictedHash
 
     # -------- CHECK DATA PROVIDERS PARAMETERS --------
 
-    :data_provider_ids,    # in input for the command, an array of DP ids
+    :data_provider_ids,    # an input for the command, a list of DP ids as string. "1,2,3"
+
+    # -------- KEEP USER IDS --------
+
+    :notify_user_id,       # an input for the command generaly the current_user
```

Comment:
Please make a comment with proper grammar and spelling: (`generally` has two Ls)

`an input for the command, generally the current user's ID`